packages/webapp/components/CookieBanner.tsx

```diff
@@ -22,7 +22,7 @@ export default function CookieBanner({
 
   return (
     <div className="flex fixed laptop:right-6 bottom-0 laptop:bottom-6 left-0 flex-col laptop:items-center laptop:p-6 py-4 pr-14 pl-4 w-full laptop:w-48 laptop:text-center laptop:rounded-2xl laptop:border border-t text-theme-label-secondary bg-theme-bg-tertiary border-theme-divider-secondary typo-footnote z-[200] laptop:left-[unset]">
-      <ModalCloseButton onClick={close} />
+      <ModalClose onClick={close} className="top-2" />
```

Comment:
I see we add `top-2` here, but no absolute position?
I see the `ModalCloseButton` actually have some inline styling for it, so maybe double check what needs to be added here.

```
      className={classNames('right-4 z-1', className)}
      style={{ position: 'absolute', ...style }}
```

Reply:
`ModalClose` is the one that won the war of the close buttons. And [it already has `!absolute` as part of the `className` prop](https://github.com/dailydotdev/apps/pull/2435/files#diff-9a6560e0e53738b292f2a629bf9396d7f35f656211f270c65cd9cbc360dbfaacR18).